alertmanager_matrix.spec

```diff
@@ -0,0 +1,56 @@
+%define debug_package %{nil}
+
+%define mybuildnumber %{?build_number}%{?!build_number:1}
+
+Name:           alertmanager_matrix
+Version:        0.0.11
```

Comment:
This should use the current git tag.

Reply:
Although there's no way to do this relatively cleanly, I have pushed a commit that works when there is a tag that describes the source.  Whether it will work or not for your users, that's your call (I know **for sure** that it _doesn't work_ in my workstation unless I add a tag first).

In normal RPM package releases, the version is written by the package author in the file, and then the change is published before tagging.